lib/presto-client/index.js

```diff
@@ -39,6 +39,7 @@ var Client = exports.Client = function(args){
     this.port = args.port || 8080;
     this.user = args.user || process.env.USER;
     this.basic_auth = args.basic_auth || null;
+    this.custom_auth = args.custom_auth;
```

Comment:
Could you add a check to make this option exclusive of `basic_auth`?

Reply:
I can add a if-statement that throws an error if both `args.custom_auth` and `args.basic_auth` exist.

In terms of the exclusiveness, I think we can move the generation of basic auth string to here and store only the `Authorization` header string to `this` (instead of storing both `basic_auth` and `custom_auth` in `this`).

What do you think @tagomoris ?

Reply:
It sounds really nice!